core/domain/services/admin/privacy_policy/PrivacyPolicy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\privacy_policy;
+
+use EEH_Template;
+use EventEspresso\core\services\privacy_policy\PrivacyPolicyInterface;
+
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
+
+
+
+/**
+ * Class PrivacyPolicy
+ * Class describes the Event Espresso core plugin's privacy policy.
+ *
+ * @package        Event Espresso
+ * @author         Mike Nelson
+ * @since          $VID:$
+ */
+class PrivacyPolicy implements PrivacyPolicyInterface
+{
+
+    /**
+     * Returns the name of the plugin and will be shown in the privacy policy's postbox header
+     * @return string
+     */
+    public function getName()
+    {
+        return esc_html__('Event Espresso', 'event_espresso');
```

Comment:
ooo actually... I missed this on my first review, but I don't think we localize 'Event Espresso' since it is a proper name. You could verify this by looking at other places it is used in the plugin and/or confirm with Seth/Garth

Reply:
I JUST had this exact conversation with WordPress core: https://core.trac.wordpress.org/ticket/43620#comment:41. But someone pointed out that translations that use other languages usually prefer to not show romantic language characters (eg Hindi, Cantonese, etc). Also, I see "Event Espresso" in tons of translated strings. But I'll bring this up with Seth

Reply:
We do localize 'Event Espresso' because of Eventsmart.